### DISCUSSION TITLE
Problema con actividad 1 Lab 05.01

### DISCUSSION THREAD

**Discussion by Bexzho** [rramosp/ai4eng.v1#47] [03 - GENERACION DE MODELOS]
Hola, buenas tardes.
No entiendo la actividad 1 del Lab 05.01.

He tratado haciendo lo mismo del ejemplo, pero la respuesta sale mala. Como lo muestro en la imagen de abajo.
![H](https://user-images.githubusercontent.com/58709463/124838453-8366e800-df4c-11eb-9608-c5e48eee33d1.png)

Profe, le pido que por favor me guie en esta actividad.
Gracias.

**Comment by Felipe-RA** [comment]
Saludos Bexzho

Necesito más información para poderte ayudar. Te agradecería si me compartes tu notebook para poder echarle un ojo directamente
